networkd/network_server.ml

```diff
@@ -148,7 +148,13 @@ let get_capabilities dev =
 		| Some vf_param ->
 			debug "enable SR-IOV on a device: %s via modprobe" dev;
 			(if enable then Modprobe.get_maxvfs driver config else Ok 0) >>= fun numvfs ->
-			if numvfs = Sysfs.get_sriov_numvfs dev then Ok Modprobe_successful
+			if numvfs = Sysfs.get_sriov_numvfs dev then
```

Comment:
So we could actually simplify the code a bit further and do 
```ocaml
[...] >>= fun numvfs ->
(* CA-287340: Even if the current numvfs equals to the target numvfs,
it is still needed to update SR-IOV modprobe config file, as the
SR-IOV enabing takes effect after reboot. For example, a user
enables SR-IOV and disables it immediately without a reboot.*)
Modprobe.config_sriov driver vf_param numvfs >>= fun _ ->
if numvfs = Sysfs.get_sriov_numvfs dev then
    Ok Modprobe_successful
else
    Ok Modprobe_successful_requires_reboot
```

Overall seems fine

Reply:
Done. Thank you :)